README.md

```diff
@@ -32,6 +32,10 @@ helm install helm-service https://github.com/keptn/keptn/releases/download/<late
 
 Replace `<latest>` with a Keptn version available from the [release list](https://github.com/keptn/keptn/tags).
 
+## Installation on multiple remote execution planes
+
+To avoid naming conflicts in a setup with multiple remote execution planes, consider providing a unique name for the service with the `nameOverride` parameter that can be found [here](./chart/README.md).
```

Comment:
I would write this even more direct. It's not like this is a suggestion, it actually doesn't work if you don't provide different names. Maybe something like
```suggestion
To avoid naming conflicts and resulting errors in a setup with multiple remote execution planes, please provide a unique name per helm-service using the `nameOverride` parameter that can be found in the [helm values](./chart/README.md).
```